### PR TITLE
workflows: add cockpit-container-update workflow

### PR DESCRIPTION
**Pull request by jelly**
This workflow tries to update our cockpit CI container on a weekly interval.

### CODE REVIEW

File: .github/workflows/cockpit-container-update.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name: cockpit-container-update
+on:
+  schedule:
+    - cron: '0 2 * * 4'
```
Comment (martinpitt) [inline .github/workflows/cockpit-container-update.yml:4]: I'd actually like to do this daily. But, does the  bot check if it already opened a PR? If not, then let's keep weekly for now.

**Review by martinpitt — APPROVED** [review]
Thanks! Question below, but not a blocker.

File: .github/workflows/cockpit-container-update.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+          printf '[user]\n\tname = Cockpit Project\n\temail=cockpituous@gmail.com\n' > ~/.gitconfig
+          echo '${{ secrets.GITHUB_TOKEN }}' > ~/.config/github-token
+
+      - name: Clone repository
+        uses: actions/checkout@v4
+
+      - name: Run cockpit-container-update
+        run: |
+          make bots
+          bots/cockpit-container-update
```
Comment (martinpitt) [inline .github/workflows/cockpit-container-update.yml:31]: It was renamed in https://github.com/cockpit-project/bots/pull/6095 , can  you please follow suit and also rename the workflow and job name?

**Review by martinpitt — CHANGES_REQUESTED** [review]

File: .github/workflows/cockpit-container-update.yml (hunk 2 above)
Comment (jelly) [reply .github/workflows/cockpit-container-update.yml:31]: Did anyone test it on a fork?

Comment (jelly) [reply .github/workflows/cockpit-container-update.yml]: And it doesn't work at all.... https://github.com/jelly/cockpit-machines/actions/runs/8329606250/job/22792320551

Comment (jelly) [reply .github/workflows/cockpit-container-update.yml]: And now it does https://github.com/jelly/cockpit-machines/actions/runs/8330370703/job/22794851154

**Review by allisonkarlitskaya — APPROVED** [review]

**Pull request merged**